flag-app/flag-app-project/src/components/CardWrapper.tsx

```diff
@@ -12,10 +12,11 @@ export const CardWrapper = ({ handleCardClick }: Props) => {
   const { countries, error, isLoading } = useCountries();
 
   if (error) return <ErrorMessage />;
+  if (!countries) return;
```

Comment:
either `return null` or return an actual component - ideally a sentence that says "No countries available or found". Returning this, which is basically `undefined`, is unfortunately not good

Reply:
yes, good idea